lint/deno.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@fresh/lint",
```

Comment:
Perhaps a new package for linting is overkill, or could perhaps allow more drift (version mismatch between `@fresh/lint` and `@fresh/core`). The lint plugin could instead be a export on `@fresh/core`, e.g. `@fresh/core/lint`.

Given that it's unlikely for lint rules to require extra dependencies, it might be better to have the lint plugin just as a new export on `@fresh/core`. I feel like most projects will want the lint rules anyway, so maybe it's better to skip another dependency.

```json
   {
     "lint": {
       "plugins": ["jsr:@fresh/core/lint"]
     }
   }
```

> [!NOTE]
> It seems that it's not possible to use import maps with `"lint.plugins"` at the moment, so requires the `jsr:` prefix currently. I created denoland/deno#30297 to highlight this.